examples/bun/src/multisig.ts

```diff
@@ -0,0 +1,71 @@
+import { wnd } from "@polkadot-api/descriptors"
+import { getMultisigSigner } from "@polkadot-api/meta-signers"
+import { sr25519CreateDerive } from "@polkadot-labs/hdkd"
+import {
+  entropyToMiniSecret,
+  mnemonicToEntropy,
+} from "@polkadot-labs/hdkd-helpers"
```

Comment:
Something to remember in the future, let's try to use [`micro-sr25519`](https://github.com/paulmillr/micro-sr25519/) in some example, just for the sake of having different libraries in them.

Reply:
I might give it a try! 👍

Reply:
Yep, sounds good 👍🏻